### PR TITLE
Populate `event_id` attributes from existing events

### PR DESCRIPTION
**Pull request by benlovell**
This is based on #438.

Simple rake tasks to populate the newly created `event_id` column via mappings from `event`.

### CODE REVIEW

File: lib/event_id_populator.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class EventIdPopulator
+  def self.populate
+    EventLog::EVENTS.each do |event|
+      EventLog.where(event: event.description, event_id: nil).update_all(event_id: event.id)
```
Comment (boffbowsh) [inline lib/event_id_populator.rb]: Will this cover all the events? Have some of the events changed their description?

Comment (benlovell) [inline lib/event_id_populator.rb]: The only renamed one I found is covered here: https://github.com/alphagov/signonotron2/pull/440/files#diff-0fe12a07e85e0e2a9814daf5085e6653R18

**Comment by benlovell** [comment]
@boffbowsh Title changed to reflect newfound mergeability! :wink:

File: db/schema.rb [hunk 2]
```diff
@@ -45,9 +45,9 @@
   add_index "batch_invitations", ["outcome"], name: "index_batch_invitations_on_outcome", using: :btree
 
   create_table "event_logs", force: :cascade do |t|
-    t.string   "uid",              limit: 255, null: false
-    t.string   "event",            limit: 255, null: false
-    t.datetime "created_at",                   null: false
+    t.string   "uid",              limit: 255,              null: false
+    t.string   "event",            limit: 255, default: "", null: false
```
Comment (boffbowsh) [inline db/schema.rb]: I don't think this is correct any more. Shouldn't be any schema changes

Comment (benlovell) [inline db/schema.rb]: :scream: rebase fail.

**Comment by benlovell** [comment]
Closing this in favour of #443 since this was targeting `revisit-event-logging`.

**Pull request closed without merging**